qlib/rl/utils/env_wrapper.py

```diff
@@ -129,9 +114,7 @@ def __init__(
         # 3. Avoid circular reference.
         # 4. When the components get serialized, we can throw away the env without any burden.
         #    (though this part is not implemented yet)
-        super().__init__()
-
-        for obj in [state_interpreter, action_interpreter, reward_fn, aux_info_collector]:
+        for obj in [reward_fn, aux_info_collector]:
```

Comment:
In the original design, (1) state interpreter, (2) action interpreter, (3) reward function, (4) auxiliary info collector, are born equal and treated equally in the EnvWrapper. This change breaks such philosophy. Extra discussion might be needed for this.

Reply:
Another reason that motivates the previous design is that I've seen some algorithms that need extra communication between components. For example, reward function might rely on extra states calculated by state interpreter to fasten its calculation. Giving them access to env wrapper makes such "hacking" possible.

Reply:
Understood. I reverted this change to let interpreters have `env` again. It's just that `env` won't be used at the moment.